source/components/Options.js

```diff
@@ -106,9 +119,9 @@ class OptionsBase extends Component<Props, State> {
   componentDidUpdate(prevProps: Props) {
     if (prevProps !== this.props) {
       if (!prevProps.isOpen && this.props.isOpen) {
-        document.addEventListener('keydown', this._handleKeyDown, false);
+        this.onOpen();
       } else if (prevProps.isOpen && !this.props.isOpen) {
-        document.removeEventListener('keydown', this._handleKeyDown, false);
+        this.onClose();
```

Comment:
Let's rename these functions to something more descriptive like `setupOnOpenListeners` and `setupOnCloseListeners`

Reply:
✅